docs/src/content/en/docs/community/contributing-templates.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Contributing Templates"
+description: "How to contribute your own templates to the Mastra ecosystem"
+---
+
+import { Callout } from "nextra/components";
+
+# Contributing Templates
+
+The Mastra community plays a vital role in creating templates that showcase innovative application patterns. This guide explains how to contribute your own templates to the Mastra ecosystem.
+
+## Template Contribution Process
+
+### 1. Review Requirements
+
+Before creating a template, ensure you understand:
+
+- [Templates Reference](/reference/templates) - Technical requirements and conventions
```

Comment:
With regard to all links. Where you can, can you link to the file "/reference/templates.mdx". This allows users to navigate the docs on GitHub.